guide/designing-products/personal-finance.md

```diff
@@ -78,12 +78,12 @@ Section structure
    layout = "float-left"
 %}
 
-This category is equivalent to how we use cash, and an often-used example is buying coffee. We typically keep small amounts of money in our wallet or purse for these types of transactions, and maybe we have planned a specific allowance for them in our monthly budgets. While some countries have fully embraced credit cards or smartphones for daily spending, other countries still prefer physical coins and bills. Many Germans, for example, prefer cash because it typically provides better privacy.
+This is the “cash”—or in this case of Bitcoin, the “sats”—that we use to make small purchases such as coffee, lunch, etc. While some countries fully embrace credit cards and smartphones for daily spending, others, such as Germany, still prefer physical coins and bills.
```

Comment:
I would keep a reference to 'why' people in Germany likes cash as it's relevant.
'..., such as Germany, still prefer cash for it's better privacy.'  or similar.

Reply:
```suggestion
This is the “cash”—or in this case of Bitcoin, the “sats”—that we use to make small purchases such as coffee, lunch, etc. While some countries fully embrace credit cards and smartphones for daily spending, others, such as Germany, still prefer cash for its better privacy.
```